### ISSUE TITLE
Clean clone does not build

### ISSUE THREAD

**Issue by layne1990** [CuppaLabs/angular2-multiselect-dropdown#229]
Did a fresh clone, npm i, and ng b and it is throwing build errors.  Breaking change in a dependency maybe?

**Comment by layne1990** [comment]
Hmm.  Must be my environment somehow.  Thanks for the quick response.  Here is a screenshot - does that error mean anything to you?
![image](https://user-images.githubusercontent.com/16440033/38892601-b6b0e428-4255-11e8-8344-9448bf84f93e.png)

**Comment by CuppaLabs** [comment]
Yes, its a mismatch of the type files in your environment. uninstall @types and npm cache clean and install again. It should solve.
